### PR TITLE
Removed dead links

### PR DESCRIPTION
**Pull request by wenzdey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1447,9 +1438,9 @@ _Note to readers: This list refers to some of the articles, posts, videos, tools
 * [Building Secure & Reliable Systems](https://www.oreilly.com/library/view/building-secure-and/9781492083115/) | [Read free online version hosted by Google](https://static.googleusercontent.com/media/sre.google/en//static/pdf/building_secure_and_reliable_systems.pdf)
 * [Site Reliability Engineering](https://www.oreilly.com/library/view/site-reliability-engineering/9781491929117/) | [Read free online version hosted by Google](https://sre.google/sre-book/table-of-contents/)
 * [The Site Reliability Workbook from Google](https://www.oreilly.com/library/view/the-site-reliability/9781492029496/) | [Read free online version hosted by Google](https://sre.google/workbook/table-of-contents/)
-* [Training Site Reliability Engineers](https://www.oreilly.com/library/view/training-site-reliability/9781492076018/) | [Read free online version hosted by Google](https://static.googleusercontent.com/media/sre.google/en//static/pdf/training-sre.pdf)
```
Comment (upgundecha) [inline README.md:1450]: Alternate link in Google GitHub Repo https://github.com/google/googlesre/blob/main/publications/Training_Site_Reliability_Engineers.pdf

File: README.md [hunk 2]
```diff
@@ -1488,7 +1479,6 @@ _Note to readers: This list refers to some of the articles, posts, videos, tools
 * [Awesome Chaos Engineering](https://github.com/dastergon/awesome-chaos-engineering)
 * [Awesome Monitoring](https://github.com/crazy-canux/awesome-monitoring)
 * [Awesome Observability](https://github.com/adriannovegil/awesome-observability)
-* [Awesome Sysadmin](https://project-awesome.org/n1trux/awesome-sysadmin)
```
Comment (upgundecha) [inline README.md:1491]: Looks like this is moved in https://github.com/awesome-foss/awesome-sysadmin

**Review by upgundecha — CHANGES_REQUESTED** [review]
Hi @wenzdey, thanks for submitting this PR. Could you please review my comments and update the PR?

**Comment by wenzdey** [comment]
> Hi @wenzdey, thanks for submitting this PR. Could you please review my comments and update the PR?

nice job finding alt links

**Comment by upgundecha** [comment]
Thanks @wenzdey 🙌

**Pull request merged**